content.md

```diff
@@ -12,47 +12,79 @@
 
 ## 1. Installation
 
-### 1.1 Tap [this link](/apks/kandu_2.0.17.apk) to download the application
-<div class="pagebreak"></div>
-
-### 1.2 Wait for the application file to finish downloading
-![screenshot](./images/1/1.jpg)
+### 1.1 From the home screen swipe down to display the notifications drawer
+![screenshot](./images/home.jpg)
 <div class="pagebreak"></div>
 
-### 1.3 Ignore the warning in the pop-up window at bottom of the screen and tap *OK*
+### 1.2 Tap the top (grey area) of the notifications drawer to expand it
 ![screenshot](./images/1/2.jpg)
 <div class="pagebreak"></div>
 
-### 1.4 Tap open on the pop-up bar on the bottom of the screen
+### 1.3 Tap the cog icon to open the device's settings menu
```

Comment:
Let's change 'cog icon' to 'settings icon (gear)' as we've decided on the other changes.